contracts/deploy/sonic/018_merkl_pool_booster.js

```diff
@@ -0,0 +1,84 @@
+const { deployOnSonic } = require("../../utils/deploy-l2");
+const addresses = require("../../utils/addresses");
+const { deployWithConfirmation } = require("../../utils/deploy.js");
+
+module.exports = deployOnSonic(
+  {
+    deployName: "018_merkl_pool_booster",
+  },
+  async ({ ethers }) => {
+    // ---------------------------------------------------------------------------------------------------------
+    // ---
+    // --- Contracts
+    // ---
+    // ---------------------------------------------------------------------------------------------------------
+    const cOSonic = await ethers.getContractAt(
+      "OSonic",
+      addresses.sonic.OSonicProxy
+    );
+
+    const cPoolBoostCentralRegistryProxy = await ethers.getContract(
+      "PoolBoostCentralRegistryProxy"
+    );
+
+    // ---------------------------------------------------------------------------------------------------------
+    // ---
+    // --- Upgrade PoolBoosterCentralRegistry
+    // ---
+    // ---------------------------------------------------------------------------------------------------------
+    const dPoolBoosterCentralRegistry = await deployWithConfirmation(
+      "PoolBoostCentralRegistry",
+      []
+    );
+    console.log(
+      `Deployed new Pool Boost Central Registry to ${dPoolBoosterCentralRegistry.address}`
+    );
+
+    const cPoolBoostCentralRegistry = await ethers.getContractAt(
+      "PoolBoostCentralRegistry",
+      cPoolBoostCentralRegistryProxy.address
+    );
+
+    // ---------------------------------------------------------------------------------------------------------
+    // ---
+    // --- Deploy PoolBoosterFactoryMerkl
+    // ---
+    // ---------------------------------------------------------------------------------------------------------
+    const hashToSign =
+      "0x97bd015d4e48fc7d8e7db116ba2d83567597ea6eb64e0694c6fccccd5a4b1841";
```

Comment:
Was this randomly generated? Or is it a hash of something?

Reply:
I copied it from this [`messageHash()`](https://sonicscan.org/address/0x8BB4C975Ff3c250e0ceEA271728547f3802B36Fd#readProxyContract) view function 28.